src/components/Editor/Resources/ResourceListSearch.vue

```diff
@@ -182,13 +199,75 @@ export default {
 					return true
 				})
 				.map(principal => {
+					const subLineData = []
+					if (principal.roomSeatingCapacity) {
+						subLineData.push(this.$t('mail', '{seatingCapacity} seats', {
+							seatingCapacity: principal.roomSeatingCapacity,
+						}))
+					}
+					if (principal.roomAddress) {
+						subLineData.push(principal.roomAddress)
+					}
+
 					return {
 						commonName: principal.displayname,
 						email: principal.email,
 						calendarUserType: principal.calendarUserType,
 						displayName: principal.displayname ?? principal.email,
+						subLine: subLineData.join(' - '),
+						// TRANSLATORS room is available due to not being booked yet
+						availability: this.$t('calendar', 'available'),
 					}
 				})
+
+			// Check room availabilities using a free busy request
+			const attendees = []
+			for (const option of options) {
+				const attendee = new AttendeeProperty('ATTENDEE', addMailtoPrefix(option.email))
+				attendees.push(attendee)
+			}
+
+			const organizer = new AttendeeProperty(
+				'ORGANIZER',
+				addMailtoPrefix(this.$store.getters.getCurrentUserPrincipalEmail),
+			)
+			const start = this.calendarObjectInstance.eventComponent.startDate
+			const end = this.calendarObjectInstance.eventComponent.endDate
+			const request = createFreeBusyRequest(start, end, organizer, attendees)
+
+			const outbox = await findSchedulingOutbox()
+			const freeBusyData = await outbox.freeBusyRequest(request.toICS())
+
+			for (const [, data] of Object.entries(freeBusyData)) {
+				if (!data.success) {
+					continue
+				}
+
+				const parserManager = getParserManager()
+				const parser = parserManager.getParserForFileType('text/calendar')
+				parser.parse(data.calendarData)
+
+				const calendarComponent = parser._calendarComponent
```

Comment:
is there a getter for the private property? the `_` is suspicous

Reply:
I just copied this section from the free busy event source. Now, the code is deduplicated and placed in its own function. Unfortunately, the parser still does not export `VFREEBUSY` (see the todo above).